fml/time/time_recorder.cc

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/fml/time/time_recorder.h"
```

Comment:
Minor Nit: Space after the related header. `clang-format` likes to reorder these headers and this tends to break the ordering rules in https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.